Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.h

```diff
@@ -51,14 +51,15 @@ class ControlGroup
 public:
   enum class CanBeDisabled
   {
-    No,
-    Yes,
+    AlwaysEnabled,
+    EnabledByDefault,
+    DisabledByDefault,
   };
 
   explicit ControlGroup(std::string name, GroupType type = GroupType::Other,
-                        CanBeDisabled can_be_disabled = CanBeDisabled::No);
+                        CanBeDisabled can_be_disabled = CanBeDisabled::AlwaysEnabled);
```

Comment:
Your suggestion works too, though I think it's more wordy. Does anyone else have an opinion on this?

Reply:
I think I'd keep the one-enum solution, but rename CanBeDisabled to something like DefaultState, so that the options become DefaultState::Enabled, DefaultState::Disabled and DefaultState::AlwaysEnabled, which is more readable IMO. Using two enums has the drawback of making it possible to specify a nonsensical combination like (CanBeDisabled::No, EnabledByDefault::No).

Reply:
I've made it use @leoetlino's suggestion.